pkg/webhook/mutation_test.go

```diff
@@ -87,6 +93,118 @@ func TestWebhookAssign(t *testing.T) {
 	}
 }
 
+func TestWebhookAssign_Conflict(t *testing.T) {
```

Comment:
Is this testing any webhook components or protocols? It looks like this may live more naturally in the mutation/system package?

Reply:
Any reason not to move this test to mutation/system, since it only relies on system packages and no webhook code?

Might improve code coverage stats, and would be good to keep around if mutation ever leaves the Gatekeeper repo.

Reply:
Removed. mutation/system already has an identical test.